src/en/administration/install_docker.md

```diff
@@ -33,13 +33,16 @@ server {
 
     location / {
         proxy_pass http://localhost:LEMMY_PORT;
+        proxy_set_header Host $host;
```

Comment:
Should this really be included in the default config? Or can it cause problems for installations that are working fine now? For example Im not sure what happens if the request already includes a Host header, will it be overwritten or appended? [The docs](https://nginx.org/en/docs/http/ngx_http_proxy_module.html#proxy_set_header) are not clear to me on this.

Reply:
If you have two `proxy_set_header` directives for the same header (not recommended) then it appends, but `proxy_set_header` can, and commonly is used to, explicitly override a default header - see http://hg.nginx.org/nginx/file/tip/src/http/modules/ngx_http_proxy_module.c#l4007.

Note also that nginx.conf for the inner proxy that splits traffic between lemmy and lemmy-ui already has this: https://github.com/LemmyNet/lemmy/blob/main/docker/nginx.conf#L68 - so when instructing people on how to expose Lemmy from the outside using another layer of proxy, I think it makes sense to have the same directive.

Reply:
Okay thanks for the contribution!